packages/react-components/src/components/VideoTile.tsx

```diff
@@ -100,27 +106,40 @@ export interface VideoTileProps {
   noVideoAvailableAriaLabel?: string;
   /** Whether the participant in the videoTile is speaking. Shows a speaking indicator (border). */
   isSpeaking?: boolean;
+
+  /* @conditional-compile-remove(PSTN-calls) */
+  /**
+   * The call connection state of the participant.
+   * For example, `Hold` means the participant is on hold.
+   */
+  participantState?: VideoGalleryRemoteParticipantState;
 }
 
 // Coin max size is set to PersonaSize.size100
 const DEFAULT_PERSONA_MAX_SIZE_PX = 100;
 // Coin min size is set PersonaSize.size32
 const DEFAULT_PERSONA_MIN_SIZE_PX = 32;
 
-const DefaultPlaceholder = (props: CustomAvatarOptions): JSX.Element => {
-  const { text, noVideoAvailableAriaLabel, coinSize, styles, hidePersonaDetails } = props;
+type DefaultPlaceholderProps = CustomAvatarOptions & {
+  participantState?: VideoGalleryRemoteParticipantState;
+};
+
+const DefaultPlaceholder = (props: DefaultPlaceholderProps): JSX.Element => {
+  const { text, noVideoAvailableAriaLabel, coinSize, hidePersonaDetails, participantState } = props;
 
   return (
     <Stack className={mergeStyles({ position: 'absolute', height: '100%', width: '100%' })}>
-      <Persona
-        styles={styles}
-        coinSize={coinSize}
-        hidePersonaDetails={hidePersonaDetails}
-        text={text ?? ''}
-        initialsTextColor="white"
-        aria-label={noVideoAvailableAriaLabel ?? ''}
-        showOverflowTooltip={false}
-      />
+      <Stack styles={defaultPersonaStyles}>
+        <Persona
+          coinSize={coinSize}
+          hidePersonaDetails={hidePersonaDetails}
+          text={text ?? ''}
+          initialsTextColor="white"
+          aria-label={noVideoAvailableAriaLabel ?? ''}
+          showOverflowTooltip={false}
+        />
+        {participantState && <Text className={mergeStyles(participantStateStyle)}>{participantState}</Text>}
```

Comment:
This text will need to be localized

Reply:
It's part of a follow up PR
Tracked by this item https://skype.visualstudio.com/SPOOL/_workitems/edit/2940251